src/cloudWatchLogs/cloudWatchLogsUtils.ts

```diff
@@ -120,4 +120,26 @@ export function findOccurencesOf(document: vscode.TextDocument, keyword: string)
     return ranges
 }
 
+const HIGHLIGHTER = vscode.window.createTextEditorDecorationType({
+    backgroundColor: new vscode.ThemeColor('list.focusHighlightForeground'),
+})
+
+export async function highlightDocument(registry: LogStreamRegistry, uri: vscode.Uri): Promise<void> {
+    const textEditor = registry.getTextEditor(uri)
+    const logData = registry.getLogData(uri)
+
+    if (!logData) {
+        throw new Error(`Missing log data in registry for uri key: ${uriToKey(uri)}. Unable to highlight`)
+    }
+
+    if (!textEditor) {
+        throw new Error(`Missing textEditor in registry for uri key: ${uriToKey(uri)}. Unable to highlight`)
```

Comment:
variable part, especially potentially long strings like a URI, should be at the very end of messages if possible. Otherwise when troubleshooting, the user sees a very long message and must visit the end to see "Unable to highlight". If "Unable to highlight" is worth mentioning, put it at the front.

```suggestion
        throw new Error(`Highlighting failed. Missing textEditor in registry for uri key: ${uriToKey(uri)}`)
```